pkg/sriov/sriov_test.go

```diff
@@ -199,6 +203,7 @@ var _ = Describe("Sriov", func() {
 			mocked := &mocks_utils.NetlinkManager{}
 
 			mocked.On("LinkByName", netconf.ContIFNames).Return(fakeLink, nil)
+			mocked.On("LinkSetAllmulticastOff", mock.Anything).Return(nil)
```

Comment:
lets stop overloading the effective mac test case,

please add separate cases (with/without restoring) for all multicast.

Reply:
ack